src/app/ui/bridge/StepsExplainer.tsx

```diff
@@ -35,89 +40,127 @@ const StepsExplainerSection = ({
-        <li> Connext</li>
-        <li>Symbiosis</li>
-        <li>Celer</li>
-      </ul>
-      <Typography variant="h4" marginY={2} sx={{ fontSize: '24px!important' }}>
-        Step 3: Select a Bridge
-      </Typography>
-      <Typography>To choose your bridge, follow these steps:</Typography>
-      <ul>
-        <li>Visualise the different quotes</li>
-        <li>
-          Check the details for each quote (i.e: amount of tokens received,
-          price impact, slippage, number of steps, gas cost, bridging time)
-        </li>
-      </ul>
-      <Typography variant="h4" marginY={2} sx={{ fontSize: '24px!important' }}>
-        Step 4: Bridge Your Assets
-      </Typography>
-      <Typography>
-        Once you have find a quote you like, you can bridge your assets from{' '}
-        {sourceToken.symbol} on {sourceChain.name} to {destinationToken.symbol}{' '}
-        on {destinationChain.name}. Follow these steps:
-      </Typography>
-      <ul>
-        <li>Click on the quote you prefer</li>
-        <li>
-          Verify the details of the quote (i.e: amount of tokens received, price
-          impact, slippage, number of steps, gas cost, bridging time)
-        </li>
-        <li>Click on "Start" execution</li>
-        <li>
-          "Approve" your tokens inside your wallet and wait for the approval
-          transaction to go through
-        </li>
-        <li>
-          "Bridge" your tokens inside your wallet and wait for the approval
-          transaction to go through
-        </li>
-      </ul>
-      <Typography variant="h4" marginY={2} sx={{ fontSize: '24px!important' }}>
-        Step 5: Verify Your Bridge
-      </Typography>
-      <Typography>
-        After bridging your assets, verify that they have been successfully
-        transferred to the {destinationChain.name} network. You can do this by
-        either:
-      </Typography>
-      <ul>
-        <li>Clicking on the buttons to see each intermediate transaction</li>
-        <li>
-          Go to your{' '}
-          <MuiLink
-            color="text.primary"
-            component={Link}
-            href="https://jumper.exchange/scan"
-          >
-            https://jumper.exchange/scan
-          </MuiLink>{' '}
-          profile to visualize your recent transaction
-        </li>
-      </ul>
+
+      <Divider />
+
+      <StepDetail
+        title={'Step 1: Prepare Your Assets'}
+        description={`Before you can bridge your assets, you need to ensure you have the
+        necessary funds and assets on the ${sourceChain?.name} network. Make sure
+        you have the correct ${sourceChain?.name} wallet address and that your
+        ${sourceToken?.name} account is funded.`}
+        img={{
+          imgUrl: `/api/widget-selection?fromToken=${sourceToken.address}&fromChainId=${sourceChain.id}&toToken=${destinationToken.address}&toChainId=${destinationChain.id}&amount=${WIDGET_AMOUNT}&theme=${theme.palette.mode}&isSwap=false`,
```

Comment:
Thing I did not see before, those imgUrl params are all duplicated, you can use `querystring` to make it clearer (and avoid duplication by declaring those at the top of the file)

Reply:
I have extracted this to first create an object[] called "steps" which takes in the necessary information for each step-item explanation. In the render message it loops through that. This seperates the data-logic from the returned render-logic. In addition with our new function "getWidgetImageProps" it should be much cleaner now.
Wdyt?